src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -749,9 +749,11 @@ void NotationViewInputController::mouseReleaseEvent(QMouseEvent* event)
     INotationNoteInputPtr noteInput = interaction->noteInput();
     const EngravingItem* hitElement = hitElementContext().element;
 
-    if (!hitElement && !m_isCanvasDragged && !interaction->isGripEditStarted()
-        && !interaction->isDragStarted() && !noteInput->isNoteInputMode()) {
-        interaction->clearSelection();
+    if (event->modifiers() != Qt::ControlModifier) {
+        if (!hitElement && !m_isCanvasDragged && !interaction->isGripEditStarted()
+            && !interaction->isDragStarted() && !noteInput->isNoteInputMode()) {
```

Comment:
I think we can combine this into one `if` condition using `&&`. Also, maybe we should do the same if the user has pressed Shift instead of Ctrl.